tests/Microsoft.Identity.Web.AotCompatibiliy.TestApp/Microsoft.Identity.Web.AotCompatibiliy.TestApp.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <TargetFrameworks Condition="'$(TargetNet9)' == 'True'">$(TargetFramework); net9.0</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+    <PublishTrimmed>true</PublishTrimmed>
+    <TrimMode>full</TrimMode>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
```suggestion
```
If we are at 0, we should keep this the default (which I assume is true).

Reply:
Done